### PR TITLE
Issue 56

### LINKED ISSUE #56: [Block] - Matterport
This block will generate the iframe required to embed the matterport content.

image

### PR DESCRIPTION
**Pull request by codevoyage**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Streamlined the embed code since the site is using only one type of embed. Plus, I removed the max-width setting to match the original site’s layout.

Fix #56 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge/tamarack-loft/10356-rockaway-ridge-st/999
- After: https://issue-56--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge/tamarack-loft/10356-rockaway-ridge-st/999

### CODE REVIEW

File: blocks/embed/embed.js [hunk 1]
```diff
@@ -1,89 +1,26 @@
...
-    block.classList = `block embed embed-${config.match[0]}`;
-  } else {
-    block.innerHTML = getDefaultEmbed(url);
-    block.classList = 'block embed';
+  block.innerHTML = getDefaultEmbed(url);
+  let embedClassList = 'block embed';
+  if (block.classList.contains('matterport')) {
+    embedClassList += ' matterport';
   }
+  block.classList = embedClassList;
```
Comment (bhellema) [inline blocks/embed/embed.js:23]: This section here seems redundant.  The block should already have the classes of 'block embed matterport' from the start of decorate.  I don't see why we need them here?

File: blocks/embed/embed.css [hunk 2]
```diff
@@ -1,20 +1,14 @@
-.embed {
+.embed.matterport {
```
Comment (bhellema) [inline blocks/embed/embed.css:1]: I'm struggling with this one _(my own mental battle)_, if the embed should know about the matterport at all.  If the matterport is used on the inventory page, then only the inventory layout should specify the styles to specify the styles specific to the matterport being used on that page?  My struggle isn't around the applied styles, its about where this style should be defined.

**Comment by bhellema** [comment]
@codevoyage can you see why the carousel images aren't showing up? https://issue-56--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge/tamarack-loft/10356-rockaway-ridge-st/999 

There is also a noticeable layout shift happening on first render.

**Comment by codevoyage** [comment]
> @codevoyage can you see why the carousel images aren't showing up? https://issue-56--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge/tamarack-loft/10356-rockaway-ridge-st/999
> 
> There is also a noticeable layout shift happening on first render.

The Word document for authoring lacks the images and is still in the old format. I need to run the importer for the inventory pages to update them to the agreed format, not sure if that was done or not already but I will do it again.

**Comment by bhellema** [comment]
Will be addressed in 123.

**Pull request closed without merging**